draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2983,23 +2743,12 @@ Completed.
 <section title="Full Implementation Procedures">
 
 <t>
-The procedures in <xref target="sec-conclude"/> require that an
-agent continue to listen for STUN requests and continue to generate
-triggered checks for a media stream, even once processing for that
-stream completes. The rules in this section describe when it is safe
-for an agent to cease sending or receiving checks on a candidate that
-was not selected by ICE, and then free the candidate.
-</t>
-
-<t>
-Once ICE processing has reached the Completed state for all peers for
-media streams using those candidates, the agent SHOULD wait an
-additional three seconds, and then it MAY cease responding to checks or
-generating triggered checks on that candidate.  It MAY free the
-candidate at that time. Freeing of server reflexive candidates is never
-explicit; it happens by lack of a keepalive.  The three-second delay
-handles cases when aggressive nomination is used, and the selected pairs
-can quickly change after ICE has completed.
```

Comment:
Seems lack of clarity on both old and new and we should tweek the new text a bit. Keep in mind forking, clients that do ICE but not media consent, etc.